plugin/trino-loki/src/main/java/io/trino/loki/LokiRecordSet.java

```diff
@@ -43,15 +44,22 @@ public LokiRecordSet(LokiClient lokiClient, LokiSplit split, List<LokiColumnHand
         }
         this.columnTypes = types.build();
 
-        // TODO: get query and start from table
-        final String query = "{source=\"stderr\"}";
-        final Long start = 1722852098L - (15 * 60 * 1000); // Start and end should be part of split
-        final Long end = 1722852098L;
+        Long end = now();
+        Long start = end - ONE_HOUR;
+        //if split.end().getEpochMillis() != 0 {}
+        //if split.start().getEpochMillis() != 0 {}
+
+        String query = "{source=\"stderr\"}";
```

Comment:
Is this a default value for the query?

Reply:
Yes. I've just hard coded it as the default.